BuildStatusChecker/Sources/BuildStatusChecker/API/Bitrise/BitriseBuild.swift

```diff
@@ -54,7 +54,11 @@ public struct BitriseBuild: Codable {
     }
 
     var groupId: String {
-        "\(commitHash)_\(parentBuildNumber ?? buildNumber)"
+        if let commitHash = commitHash {
+            return "\(commitHash)_\(parentBuildNumber ?? buildNumber)"
+        } else {
+            return "\(parentBuildNumber ?? buildNumber)"
+        }
```

Comment:
CommitHash can be optional which would result in "Optional<String>(nil)" in the string.